pkg/controller/account/account_controller.go

```diff
@@ -169,6 +166,25 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 			return reconcile.Result{}, err
 		}
 
+		awsSetupClient, err := r.awsClientBuilder.GetClient(controllerName, r.Client, setupClientInput)
+		if err != nil {
+			reqLogger.Error(err, "failed building operator AWS client")
+			if aerr, ok := err.(awserr.Error); ok {
+				switch aerr.Code() {
+				// If it's AccessDenied we want to just delete the finalizer and continue as we assume
```

Comment:
Might be useful to put some logging here to help improve visibility on this - we can remove at a later date if found unnecessary

Reply:
the code on line 171 is already logging the error, the `switch` code just allows us to define if it's specifically an AWS error or not.  The "print" version of the AWS error ends up looking something like this in the logs:

`AccessDenied: User: arn:aws:iam::REDACTED:user/REDACTED is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::REDACTED:role/REDACTED\n\tstatus code: 403, request id: 14258866-1552-11ec-82a8-0242ac130003`

Is there something specific you're looking for in the logs you're thinking about that's missing?

Reply:
Nope that seems fair - just wanted to double check to see if we wanted to have it highlighted in our logs when a customer removes our creds and therefore we can't clean up correctly after ourselves